x-pack/legacy/plugins/apm/public/components/shared/TransactionActionMenu/TransactionActionMenu.tsx

```diff
@@ -100,7 +100,7 @@ export const TransactionActionMenu: FunctionComponent<Props> = ({
     <>
       {isCustomLinkFlyoutOpen && (
         <CustomLinkFlyout
-          customLinkSelected={{ filters: { ...filters } } as CustomLinkType}
+          customLinkSelected={{ filters } as CustomLinkType}
```

Comment:
Why use a type cast here? Are props of `CustomLinkFlyout` defined as required that should actually be optional?

Reply:
I'm also unsure about L64:71, where we are using `as Filter[]`, when the items created do not match the type signature of `Filter` (as `value` is `string | undefined` but should be just `string`).